pkgs/cupertino_http/README.md

```diff
@@ -4,11 +4,19 @@
 A macOS/iOS Flutter plugin that provides access to the
 [Foundation URL Loading System][].
 
-## Experiment warning
+## Status: Experimental
 
-**NOTE**: This package is experimental, and shared for feedback.
-Breaking changes may be made without announcement.
-The whole package may get discontinued.
+**NOTE**: This package is currently experimental and published under the
+[labs.dart.dev](https://dart.dev/dart-team-packages) pub publisher in order to
+solicit feedback. 
+
+For packages in the labs.dart.dev publisher we generally plan to either graduate
+the package into a supported publisher (dart.dev, tools.dart.dev) after a period
+of feedback and iteration, or discontinue the package. These packages have a
+much higher expected rate of API and breaking changes.
+
+Your feedback is valuable and will help us evolve this package. For general
+feedback, suggestions, and comments, please file an issue in the bug tracker.
```

Comment:
Could you make "bug tracker" a link to the bug tracker?

Reply:
Certainly, done!